File: trunk/JCodeConverterLite/logMio.js


//#include "SFParameter.h";


//SFParameter::SFParameter() {
//this->type=GLOBAL_GENERIC;
//}


function SFParameter( name) {
	this.name = name;
}

function SFParameter( name,type) {
	this.name = name;
	this.type = type;
}

SFParameter.prototype["getName"]=function(){
	return this.name;
};

SFParameter.prototype["getType"]=function(){
	return this.type;
};

SFParameter.prototype["setType"]=function(type){
	this.type=type;
}

File: trunk/JCodeConverterLite/logMio.cpp
#include SFParameter.h
#include <iostream>

//package shadow.pipeline.parameters;




//protected short type=GLOBAL_GENERIC;


SFParameter::SFParameter( ):SFParameteri() {
	this->ancheMe=false;
}

SFParameter::SFParameter( string name,short type):SFParameteri() {
	this->name = name;
	this->type = type;
	this->ancheMe=false;
}

SFParameter::SFParameter( string name):SFParameteri() {
	this->name = name;
	this->ancheMe=false;
}

string SFParameter::getName( ) {
	return this->name;
}

short SFParameter::getType( ) {
	return this->type;
}

void SFParameter::setType (short type ) {
	this->type=type;
}

//@Override	public String toString() {
//return name+"("+type+")";
//}


